core/src/main/java/org/openedx/core/presentation/dialog/IAPDialogFragment.kt

```diff
@@ -228,6 +225,10 @@ class IAPDialogFragment : DialogFragment() {
         }
     }
 
+    override fun getTheme(): Int {
```

Comment:
My status bar color is this now.

<img src="https://github.com/user-attachments/assets/d1db337d-ee13-4437-a7e6-b0674ab28461" height=300/>